### PR TITLE
chore(proxy): bump link

### PR DESCRIPTION
**Pull request by xla**
Signed-off-by: Alexander Simmerl <a.simmerl@gmail.com>

### CODE REVIEW

File: proxy/coco/src/config.rs [hunk 1]
```diff
@@ -60,13 +60,15 @@ where
...
+            advertised_addrs: None,
+            fetch: Default::default(),
             paths,
             listen_addr,
             membership: net::protocol::membership::Params::default(),
             network: net::Network::default(),
             replication: replication::Config::default(),
         },
-        storage_pools: net::peer::PoolSizes::default(),
+        storage: Default::default(),
```
Comment (FintanH) [inline proxy/coco/src/config.rs:71]: You've changed your tune :stuck_out_tongue_closed_eyes:

File: proxy/coco/src/state.rs [hunk 2]
```diff
@@ -163,14 +163,15 @@ where
...
-                store,
-                config,
-                Some(owner),
+            let fetcher = librad::git::storage::fetcher::PeerToPeer::new(
                 urn.clone(),
                 remote_peer,
                 addr_hints,
             )
+            .build(&store)
+            .unwrap()
```
Comment (FintanH) [inline proxy/coco/src/state.rs:172]: I made some changes in `daemon` that might be worth porting over https://github.com/radicle-dev/radicle-link/pull/576/commits/3484e241fadaa9e3109182c48a27a6fea8e014df

Comment (xla) [reply proxy/coco/src/state.rs]: 052509d

**Review by FintanH — DISMISSED** [review]

**Comment by xla** [comment]
Superseded by #1829

**Pull request closed without merging**